libs/gdk/src/lib/gdk-json-input/gdk-json-input.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, forwardRef, OnDestroy, Output } from '@angular/core';
+import { FormControl, NG_VALIDATORS, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { MatFormFieldControl } from '@angular/material/form-field';
+import { EventEmitter } from 'events';
+import { Subject } from 'rxjs';
+
+@Component({
+  selector: 'gdk-json-input',
+  templateUrl: './gdk-json-input.component.html',
+  styleUrls: ['./gdk-json-input.component.scss'],
+  providers: [
+    { provide: NG_VALUE_ACCESSOR, useExisting: forwardRef(() => GdkJsonInputComponent), multi: true },
+    { provide: NG_VALIDATORS, useExisting: forwardRef(() => GdkJsonInputComponent), multi: true },
+    { provide: MatFormFieldControl, useExisting: GdkJsonInputComponent }
+  ]
+})
+export class GdkJsonInputComponent extends MatFormFieldControl<any> implements OnDestroy {
+  @Output() valueChanged = new EventEmitter();
+  public jsonString: string;
+  public stateChanges = new Subject<void>();
+  private parseError: boolean;
+  private data: any;
+
+  public writeValue(obj: any) {
+    if (obj) {
```

Comment:
What if `obj` is `{ }`, would that be a problem?

Reply:
the { } is a valid json.. so, no..